src/pages/using-spark/guides/alt-text.mdx

```diff
@@ -0,0 +1,352 @@
+---
+title: Alternative Text
+---
+
+import { SprkDivider, SprkIcon } from '@sparkdesignsystem/spark-react';
+
+<h1 class="sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
+  Alternative Text
+</h1>
+
+UX Writers, designers and developers can use this **Alternative Text Guide** as an instruction to writing style, usage, text for copy, documentation, reference information, and copy for all images inside Rocket Products digital experiences.
+
+<SprkDivider element="span" additionalClasses="sprk-u-mvl" />
+
+<h3 class="sprk-b-TypeDisplayThree sprk-u-mbm sprk-u-Measure">Main Takeaways</h3>
+
+- Every image's alternative text should describe the image without altering the meaning of the screen or referencing the word “image.”
+- When writing copy for alternative text, imagine what you would say when reading the screen to someone over the phone without mentioning that there is an image on the screen.
```

Comment:
`When writing alternative text, consider how you would describe that image to someone without them having to visually see the image.`

This sentence is more more inclusive of abilities this way.

Reply:
When writing alternative text, consider how you would describe the image to someone _with a visual impairment._ @RVMendoza let me know your thoughts.

Reply:
I think categorizing it as solely for people with visual impairments isn't inclusive to what we're saying. That's why I used "someone that isn't able to see it visually"  -- just like captions on videos, alt text gives access to more people, and to situations we may not even think of.

We're saying that no matter what (doesn't matter what impairment, be it visual, or someone who isn't using a visual device), you need to be able to describe it well to them.